src/connectedvmware/azext_connectedvmware/_params.py

```diff
@@ -338,20 +340,78 @@ def load_arguments(self, _):
 
     with self.argument_context('connectedvmware vm guest-agent enable') as c:
         c.argument(
-            'vm_name', options_list=['--vm-name'], help="Name of the VM."
+            'vm_name', help="Name of the VM."
         )
         c.argument(
             'username',
             options_list=['--username'],
             help="Username to use for connecting to the vm.",
         )
         c.argument(
-            'password',
-            options_list=['--password'],
+            'password', options_list=['--password'],
             help="Username password credentials to use for connecting to the VM.",
         )
 
     with self.argument_context('connectedvmware vm guest-agent show') as c:
         c.argument(
-            'vm_name', options_list=['--vm-name'], help="Name of the VM.",
+            'password', options_list=['--password'],
+            help="Username password credentials to use for connecting to the VM.",
         )
+
+    with self.argument_context('connectedvmware vm guest-agent show') as c:
+        c.argument(
+            'vm_name', help="Name of the VM.",
+        )
+
+    with self.argument_context('connectedvmware vm extension list') as c:
+        c.argument('vm_name', type=str, help='The name of the vm containing the extension.')
+        c.argument(
+            'expand', help='The expand expression to apply on the operation.',
+            deprecate_info=c.deprecate(hide=True))
```

Comment:
If this parameter is not in the old code, you don't need to add it to this new PR. 
It is added in the sample PR because this parameter exists in the old code, so the deprecated information is added to avoid breaking change to users

Reply:
As per discussion make the code changes.